SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -84,3 +86,35 @@ void FSpatialGDKEditor::GenerateSnapshot(UWorld* World, FString SnapshotFilename
 		FailureCallback.ExecuteIfBound();
 	}
 }
+
+void FSpatialGDKEditor::LaunchCloudDeployment(FSimpleDelegate SuccessCallback, FSimpleDelegate FailureCallback)
+{
+	LaunchCloudResult = Async<bool>(EAsyncExecution::Thread, SpatialGDKCloudLaunch,
```

Comment:
Did you consider using a FRunnable? https://api.unrealengine.com/INT/API/Runtime/Core/HAL/FRunnable/index.html not sure what is preferable

Reply:
That said, this approach is probably good too

Reply:
Haven't looked into it - decided to implement it similarly to the way the schema generation works so that it's consistent with the rest of the code: https://github.com/spatialos/UnrealGDK/blob/928670f932571341c8ea8572419eb00d7542615b/SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp#L50